src/test/kotlin/no/skatteetaten/aurora/gobo/resolvers/errorhandling/GoboDataFetcherExceptionHandlerTest.kt

```diff
@@ -22,27 +22,29 @@ class GoboDataFetcherExceptionHandlerTest {
     @Test
     fun `Given GoboException add GraphQL error to execution context`() {
         val handlerParameters = DataFetcherExceptionHandlerParameters(
-                executionContext,
-                null,
-                Field("name"),
-                null,
-                null,
-                null,
-                GoboException("test exception"))
+            executionContext,
+            null,
+            Field("name"),
+            null,
+            null,
+            null,
+            GoboException("test exception")
+        )
         exceptionHandler.accept(handlerParameters)
         verify { executionContext.addError(any()) }
     }
 
     @Test
     fun `Given Exception do not add GraphQL error`() {
         val handlerParameters = DataFetcherExceptionHandlerParameters(
-                executionContext,
-                null,
-                Field("name"),
-                null,
-                null,
-                null,
-                Exception("test exception"))
+            executionContext,
```

Comment:
Kan vi legge på named parameters her?

Reply:
Det syns jeg er et godt forslag, men ser ikke ut til at jeg får gjort det:  
`Named arguments are not allowed for non-Kotlin functions`